docs/contributing/managing-pull-requests.md

```diff
@@ -171,8 +171,9 @@ Every PR opened in the repository needs to be approved before it can be merged.
 
 Typically this is:
 
-- **gatsbyjs/core** for Code
-- **gatsbyjs/learning** for Documentation
+- **gatsbyjs/themes** for certain features like Themes, Recipes, and Admin
+- **gatsbyjs/learning** for documentation
+- **gatsbyjs/core** for any other code
```

Comment:
Add https://github.com/orgs/gatsbyjs/teams/website?

Reply:
Maybe? I don't know if there is a workflow around that team or not.